### DISCUSSION TITLE
Desktop measure tool not working?

### DISCUSSION THREAD

**Discussion by juanafernand** [devemux86/cruiser#265] [Q&A] [answered]
It seems that Measure Tool is not working properly on Windows when trying to drag start/end point.
Could you check it?

**Comment by devemux86** [comment, marked as answer]
@juanafernand the drag of markers on desktop requires an extra key:
- **Space** or **Caps Lock** + mouse drag

All desktop keyboard shortcuts are described in the Wiki:
https://wiki.openstreetmap.org/wiki/Cruiser#Keyboard_shortcuts

**Reply by juanafernand** [reply]
Thanks!
